docs/services.md

```diff
@@ -1,101 +1,99 @@
+   When the Fabric peer commits a token transaction `Tx`, the FSC node gets informed.
 2. The `Delivery Client` informs the `Committer` that `Tx` is available.
 3. If `Tx` is deemed valid by Fabric, then the `Committer` further manipulates the RWSet by invoking all registered
-   `RWSet Processors` before the RWSet gets committed into the vault. If `Tx` is invalid, the transaction gets directly
+   `RW Set Processors` before the RW Set gets committed into the vault. If `Tx` is invalid, the transaction is
    discarded.
 4. The Token SDK installs one of these processors, the `Token RW Set Processor`. This processor extracts all the token
-   related information and augments the RWSet with additional key-value pairs to speed up the
-   token selection process, among other things.
-5. Finally, the RWSet, after processing, is committed to the vault.
-
-Only at this point, the tokens created by the transaction become available inside the Token Vault and can be selected
-for a new transactions.
-
-## The Token Namespace and Chaincode
-
-In the previous Section, we have seen that part of the lifecycle of a Token Transaction is the invocation of the
-`Token Chaincode` that is associated to the `Token Namespace`.
-
-The `Token Namespace` contains the tokens and any additional information needed to make sense out of them
-in a key-value format. Attached to the namespace we have:
-- The [`Token Chaincode`](https://github.com/hyperledger-labs/fabric-token-sdk/tree/main/token/services/network/fabric/tcc),
-  that exposes functionalities useful to develop token applications, and
-- The `Endorsement Policy` that describe the governance of the token chaincode. In other words, who is allowed to
-  modify the namespace.
-
-The Token Chaincode must be initialized with some `public parameters`.
-The public parameters depends on the specific token driver implementation and can be generated by
-using the utility [`tokengen`](./../cmd/tokengen/README.md).
-
-Once initialized, the token chaincode can provide the following functionalities:
-- `Fetch the public parameters`. They must be fetched by each FSC node running the Token SDK stack.
-  This is done automatically.
-- `Fetch Tokens` is used to retrieve the content of tokens by their ids.
-- `Validate and Translate Token Requests`. This is one of the essential steps in the lifecycle of a token transaction,
-  as we have seen in the previous section.
\ No newline at end of file
+   related information and augments the RW Set with additional key-value pairs to speed up the token selection process, among other things.
+5. Finally, the RW Set, after processing, is committed to the vault.
+
+Only at this point, the tokens created by the transaction become available via the Token Vault service we have discussed above.
+
+### Orion Driver
+
+The Orion driver is similar to the Fabric driver because also Orion manages RW Sets.
+Though, in Orion there is not concept of chaincode or stored routing. 
```

Comment:
Orion does not support the concept of chaincode nor stored routing. What is stored routing?

Reply:
stored routines, sorry :)